### PR TITLE
Mark Elastica 5.x as unmaintained

### PR DESCRIPTION
**Pull request by thePanz**
ES v5.6.x EOL was on 2019-03-11

### CODE REVIEW

**Comment by ruflin** [comment]
@thePanz Yes, feels good to have it in the docs. Even though this is not directly release related, I think it should show up in the Changelog that we marked it so people are aware. Don't have a better place for it unfortunately.

**Comment by thePanz** [comment]
I agree @ruflin ! I will open another MR on the 5.x branch. wdyt?

**Comment by ruflin** [comment]
@thePanz I think both would be great, as the Changelog users see when announcing a new release is the "current" on. Suggestion: Lets not put this info into a specific section but directly under "Unreleased" for now: https://github.com/ruflin/Elastica/blob/master/CHANGELOG.md#unreleased

**Comment by ruflin** [comment]
@thePanz In case you missed the ping here ;-)

**Comment by thePanz** [comment]
@ruflin new changes pushed! :)

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ All notable changes to this project will be documented in this file based on the
 ### Deprecated
 
 ## [Unreleased](https://github.com/ruflin/Elastica/compare/7.0.0-beta1...7.0.0-beta2)
+## [Unreleased](https://github.com/ruflin/Elastica/compare/7.0.0-beta1...master)
```
Comment (ruflin) [inline CHANGELOG.md]: Rebase issue?

**Comment by ruflin** [comment]
LGTM except the duplication of the line ;-)

**Comment by thePanz** [comment]
image
Well, the changelog for the 7.0.0-beta2 has an issue ;-) they're both "unreleased" :)

**Comment by thePanz** [comment]
@ruflin I fixed that issue too :)

**Comment by ruflin** [comment]
Thanks for the fix(es)!

**Pull request merged**